src/Language/PureScript/CodeGen/JS/Optimizer.hs

```diff
@@ -77,7 +77,8 @@ optimize' js = do
     , inlineOperator (C.prelude, (C.$)) $ \f x -> JSApp f [x]
     , inlineOperator (C.prelude, (C.#)) $ \x f -> JSApp f [x]
     , inlineOperator (C.preludeUnsafe, C.unsafeIndex) $ flip JSIndexer
-    , inlineCommonOperators ]) js
+    , inlineCommonOperators
+    , inlineAppliedArrComposition ]) js
```

Comment:
Would this be the right place to put this optimisation? Is there a better order?